ui/app/src/components/ItemDisplay/utils.tsx

```diff
@@ -70,6 +82,39 @@ export function getItemStateText(stateMap: ItemStateMap, values?: Record<string,
 	);
 }
 
+export function getItemStateString(
```

Comment:
@rart I created 2 utils to return a string for the itemState and itemPublishingTarget, since the ones used in ItemDisplay return `<FormattedMessage...` />. Should we replace the other utils and only use the ones that return string?